crates/noirc_evaluator/src/brillig/acvm_brillig.rs

```diff
@@ -0,0 +1,155 @@
+use acvm::FieldElement;
+//THIS IS A TEMPORARY FILE, all warnings are disabled.
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+
+pub type Label = usize;
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all)]
+pub type BrilligOpcode = Opcode;
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+pub type BrilligType = Typ;
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+pub type BrilligValue = Value;
+
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+#[derive(Debug, Clone, PartialEq, Eq, Copy)]
+pub enum RegisterValueOrArray {
+    RegisterIndex(RegisterIndex),
+    HeapArray(RegisterIndex, usize),
+}
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum Opcode {
```

Comment:
Yep, this is here since we don't have an acvm version with Brillig in Noir yet. So this PR plus any Brillig related PR's would not be mergable without a stub